js-arcade/tictactoe/__test__/TicTacToeSpec.js

```diff
@@ -97,5 +97,36 @@ describe("TicTacToe", function() {
         });
     });
     describe("isGameOver", function() {
+        it("does not end game with no tic tac toe", function () {
+            expect(game.isGameOver(0,0)).toBe(false)
+        })
+        it("ends game with a row of x's ", function () {
+            game.squares[0][0] = true
+            game.squares[0][1] = true
+            game.squares[0][2] = true
+
+            expect(game.isGameOver(0,3)).toBe(true)
+        })
+        it("ends game with a column of x's ", function () {
+            game.squares[0][0] = true
+            game.squares[1][0] = true
+            game.squares[2][0] = true
+
+            expect(game.isGameOver(0,0)).toBe(true)
+        })
+        it("ends game with a diagonal of x's ", function () {
+            game.squares[0][0] = true
+            game.squares[1][1] = true
+            game.squares[2][2] = true
+
+            expect(game.isGameOver(0,0)).toBe(true)
+        })
+        it("ends game with a cross-diagonal of x's ", function () {
+            game.squares[0][2] = true
+            game.squares[1][1] = true
+            game.squares[2][0] = true
+
+            expect(game.isGameOver(2,0)).toBe(true)
+        })
```

Comment:
Don't forget negative scenarios too! e.g. a row of mixed true/false, to make sure you're not just checking that they're filled or something - but that they actually match. When the game is not over is just as important - you don't want the game ending prematurely.